Client/src/components/pages/Dashboard/AdminDashboard/index.jsx

```diff
@@ -121,7 +121,19 @@ const Index = () => {
         <Link to="/admin-create-products" className="inline-block px-4 py-2 bg-green-500 text-white rounded hover:bg-green-600 mr-20">+ ADD PRODUCT</Link>
       </div>
 
-      <Table data={paginateRecords} type="product" handleUpdate={handleUpdate} handleProductDelete={handleProductDelete} />
+      {/* <Table data={paginateRecords} type="product" handleUpdate={handleUpdate} handleProductDelete={handleProductDelete} /> */}
+      <Table
+        data={paginateRecords}
+        headers={[
+          { key: 'id', label: 'Product ID' },
```

Comment:
same as above refactor it

Reply:
Okay, Noted, Applying the changes and Re-pushing the code.